builder/builder-module.c

```diff
@@ -1343,9 +1372,10 @@ builder_module_build (BuilderModule  *self,
         }
     }
 
-  has_configure = g_file_query_exists (configure_file, NULL);
+  if (!simple)
+    has_configure = g_file_query_exists (configure_file, NULL);
 
-  if (!has_configure && !self->no_autogen)
+  if (!simple && !has_configure && !self->no_autogen)
```

Comment:
Yes, thats what i meant. We always do in order:
1) autogen
2) configure (or equivalent)
3) make (or equivalent)
4) build-commands
5) make install (or equivalent)
6) post-install

The build-system type affects the "or equivalent", and setting simple just means skip these.